packages/backend/src/helpers/auth.ts

```diff
@@ -61,6 +65,30 @@ export async function getOrCreateUser(email: string): Promise<User> {
   return user
 }
 
+export async function sendOnboardingEmail(user: User) {
+  // check if user has logged in before and has been created
+  // after the specified date for the release of onboarding email
+  if (
+    user.lastLoginAt !== null ||
+    new Date(user.createdAt) < ONBOARDING_EMAIL_RELEASE_DATE
+  ) {
+    return
+  }
+  // call plumber webhook to send onboarding email only in prod
+  try {
+    if (appConfig.isProd && appConfig.onboardingEmailWebhookUrl) {
```

Comment:
why only for prod environment?

also do you want to use launchdarkly flag instead of another env var 🤔

Reply:
i thought it was unnecessary to send email for non prod envt but i guess its easier to test and switch it off if we use LD flag